.github/component_owners.yml

```diff
@@ -57,6 +57,8 @@ components:
   src/OpenTelemetry.Instrumentation.Runtime/:
     - twenzel
     - xiang17
+  src/OpenTelemetry.Instrumentation.ServiceFabricRemoting/:
+    - sablancoleis
```

Comment:
Is there any chance to have second volunteer to help with the maintenance?

Reply:
I have been asking the Service Fabric Team for volunteers but so far no one has. I will update the owners if find somebody.